config/locales/blacklight.ar.yml

```diff
@@ -111,6 +111,9 @@ ar:
         sort:
           count: ترتيب رقمي
           index: ترتيب أبجدي
+        suggest:
```

Comment:
Is it expected that these be in the target language? Or is it ok to start with English and then have a future edit with the real translation?

Reply:
Thanks for pointing this out.  It looks like I used the [`add_missing` task from i18n-tasks](https://github.com/glebm/i18n-tasks?tab=readme-ov-file#add-missing-keys), when I should have used [`translate_missing`](https://github.com/glebm/i18n-tasks?tab=readme-ov-file#translate-missing-keys), which grabs a preliminary translation from Google Translate.